src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -208,6 +208,26 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
     }
 
+    void TermControl::SearchNextMatch()
+    {
+        if (_searchBox->TextBox().Text().empty())
+        {
+            return;
+        }
+        keyBindingSearch = true;
+        _Search(_searchBox->TextBox().Text(), true, false);
+    }
+
+    void TermControl::SearchPreviousMatch()

```

Comment:
Would TermControl::SearchPrevMatch() be a better name as would match all the other occurences of "Prev"?